cms/static/sass/views/_container.scss

```
@@ -239,6 +239,17 @@
             color: $gray-l1;
           }
         }
+
+        .action-copy {
+          width: 100%;
+          border-color: #0075b4;
+          padding-top: 12px;
+          padding-bottom: 12px;
```

```suggestion
          padding-top: 10px;
          padding-bottom: 10px;
          line-height: 24px;
```
@pkulkark This will fix the issue where it moves around _slightly_ even on devstack. Still not sure why the `padding` rules aren't showing up on the sandbox though...